wallet/walletrpc.c

```diff
@@ -1243,14 +1243,15 @@ static struct command_result *json_reserveinputs(struct command *cmd,
 	struct unreleased_tx *utx;
 	const struct utxo **chosen_utxos;
 	struct bitcoin_tx_output **outputs;
-	u32 *feerate_per_kw, *minconf;
+	u32 *feerate_per_kw, *minconf, *expire_at;
```

Comment:
Just to match the param name:

```suggestion
	u32 *feerate_per_kw, *minconf, *expire_after;
```